core/src/transport/mod.rs

```diff
@@ -54,6 +54,36 @@ pub use self::memory::connector;
 pub use self::muxed::MuxedTransport;
 pub use self::upgrade::UpgradedNode;
 
+
+// This is a new error type that you've created. It represents the ways a
```

Comment:
Is this copy-pasted from a tutorial? :sweat_smile:

Reply:
Yes. Forgot to remove it after. Sorry about that.